### ISSUE TITLE
Allowing get method but not list

### ISSUE THREAD

**Issue by florisporro** [portable-cto/airtable-proxy-worker#6]
First off, great package! Super happy I found this as it's exactly what I needed.

It would be great if there was a way to prohibit a list operation, but still allow get of individual resources.

**Comment by karllhughes** [comment]
Good idea. I'm not actively working on this project much, but open to PRs if you want to give it a shot!
